zmq/tests/test_message.py

```diff
@@ -279,7 +274,7 @@ def test_buffer_numpy(self):
             shape = shapes[:i]
             A = numpy.random.random(shape)
             m = zmq.Frame(A)
-            if view.__name__ == 'buffer':
+            if memoryview.__name__ == 'buffer':
```

Comment:
This block can be removed

Reply:
I wrote on that commit: _There is opportunity to remove tests but let's make sure that these changes pass the current tests before removing any of them._